components/dashboard/src/dedicated-setup/SSOSetupStep.tsx

```diff
@@ -93,9 +93,13 @@ export const SSOSetupStep: FC<Props> = ({ config, onComplete }) => {
             <div className="mb-10">
                 <Heading1>Configure single sign-on</Heading1>
                 <Subheading>
-                    {/* TODO: Find what link we want to use here */}
                     Enable single sign-on for your organization using the OpenID Connect (OIDC) standard.{" "}
-                    <a href="https://openid.net/connect/" target="_blank" rel="noreferrer noopener" className="gp-link">
+                    <a
+                        href="https://gitpod.io/docs/dedicated/sso-setup-8f7696e0"
```

Comment:
**thought**: Not sure why we have the alphanumeric suffix at the end of the link, but the link works as expected.

Cc @lucasvaltl @easyCZ @Siddhant-K-code because https://github.com/gitpod-io/website/pull/3633

Reply:
It was answered [here](https://gitpod.slack.com/archives/C01KGM9ETU6/p1683886093576519) (Internal conversation)